### PR TITLE
Implementation electrochemistry units:

### PR DESCRIPTION
**Pull request by JAC28**
Implementation of the units:
- mAh/g
- A/g 
which are commonly used in electrochemistry for standardization of test conditions. 
For implementation quantity kinds
- specific electric current
- specific electric charge 
and corresponding dimensions vectors were added.

### CODE REVIEW

File: vocab/quantitykinds/VOCAB_QUDT-QUANTITY-KINDS-ALL-v2.1.ttl [hunk 1]
```diff
@@ -4676,6 +4683,13 @@ quantitykind:ElectricCurrent
   rdfs:isDefinedBy <http://qudt.org/2.1/vocab/quantitykind> ;
   rdfs:label "Electric Current"@en ;
 .
+quantitiykind:SpecificElectricCurrent
```
Comment (steveraysteveray) [inline vocab/quantitykinds/VOCAB_QUDT-QUANTITY-KINDS-ALL-v2.1.ttl:4686]: Typo: Should be quantitykind:

File: vocab/quantitykinds/VOCAB_QUDT-QUANTITY-KINDS-ALL-v2.1.ttl [hunk 2]
```diff
@@ -28,6 +28,13 @@
   owl:imports <http://qudt.org/2.1/vocab/dimensionvector> ;
   owl:versionIRI <http://qudt.org/2.1/vocab/quantitykind> ;
 .
+quantitykind:SpecificElectricCharge
+  rdf:type qudt:QuantityKind ;
+  qudt:hasDimensionVector <http://qudt.org/vocab/dimension/A0E1L0I0M-1H0T1D0> ;
```
Comment (steveraysteveray) [inline vocab/quantitykinds/VOCAB_QUDT-QUANTITY-KINDS-ALL-v2.1.ttl:33]: Apologies - I need to update our wiki documentation. Please replace with:

qudt:hasDimensionVector qkdv:A0E1L0I0M-1H0T1D0 ;

**Review by steveraysteveray — CHANGES_REQUESTED** [review]
Very thorough submission! Thanks.

**Review by steveraysteveray — CHANGES_REQUESTED** [review]
Very thorough submission! Thanks.

**Comment by JAC28** [comment]
@steveraysteveray thank you for the review, I hope I got it right now :)

**Review by steveraysteveray — APPROVED** [review]

**Pull request merged**